### PR TITLE
Help markup

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

File: src/main/resources/hudson/tasks/Mailer/global.jelly [hunk 1]
```diff
@@ -32,8 +32,7 @@ THE SOFTWARE.
       <f:textbox />
     </f:entry>
     <f:advanced>
-      <f:optionalBlock name="useSMTPAuth" title="${%Use SMTP Authentication}" checked="${descriptor.smtpAuthUserName!=null}"
-          help="${descriptor.getHelpFile('smtpAuth')}">
```
Comment (jsoref) [inline src/main/resources/hudson/tasks/Mailer/global.jelly]: I'm pretty sure that this line is causing a 404 when I look at jenkins/configure...

I'm not as certain about the fix...

**Comment by oleg-nenashev** [comment]
So you just remove the help here, right? Would it be possible to add a static help file instead (if needed)?

**Comment by jsoref** [comment]
Maybe, I don't know enough about that.

**Comment by jsoref** [comment]
@oleg-nenashev: that sounds about right, but I think if someone wants to do that, they should make a distinct pull request to provide that content plus the hook.

**Comment by alecharp** [comment]
I'm closing this as there is no activity on the pull request and its content is not complete.

**Pull request closed without merging**